metricbeat/helper/prometheus/textparse.go

```diff
@@ -568,8 +577,8 @@ func ParseMetricFamilies(b []byte, contentType string, ts time.Time) ([]*MetricF
 			var counter = &Counter{Value: &v}
 			mn := lset.Get(labels.MetricName)
 			metric = &OpenMetric{Name: &mn, Counter: counter, Label: labelPairs}
-			if isTotal(metricName) && contentType == OpenMetricsType { // Remove suffix _total, get lookup metricname
```

Comment:
Is this comment no longer valid? If so why?

Reply:
It is, I must have removed it by accident. I will add it again.